airflow/plugins/generate_airtable_mapping_table.py

```diff
@@ -0,0 +1,202 @@
+# This is a little helper to generate SqlToWarehouse operator tasks
+# to create mapping tables for Airtable data
+
+# To run, start IPython within the airflow/plugins directory:
+# `python3 -m IPython` or similar
+# `import generate_airtable_mapping_table`
+# `generate_airtable_mapping_table.process_csv(
+#   '<path to CSV file>',
+#  '<path to directory to write outputs
+#   (should probably be airflow/dags/airtable_views)>')`
+
+# the CSV file should have four fields: table1, table2, col1, col2
+# where table1 and table2 are warehouse (not raw Airtable) table names
+# and col1 and col2 are the corresponding (warehouse) column names
+# that define the linked record relationship
+# in cases of self-joins, leave table2 and col2 blank
+# and just define the table in table1 and the foreign key column in col1
+# (for example, services.paratransit_for)
+
+import pandas as pd
+
+
+def generate_sql(table1, table2, col1, col2):
```

Comment:
In the times I have seen code used to generate code that is then copied and pasted elsewhere to be executed it is a red flag of code that probably shouldn't exist in that form. This is an anti-pattern for the following reason in my opinion:

- It creates two codebases that need to be maintained (one for the code that generates the code and another for the other code that is generated)
- The generated code is not always easily testable in various fashions (in execution, static typing or via automated testing)
- It requires extra build steps that are not needed had there been just one codebase

This appears to be trying to make the text for `SqlToWarehouseOperator` DAG tasks. A better approach for this is to refactor this file into two new operators:

1. AirtableTwoTableJoinOperator
2. AirtableSelfJoinOperator

In doing this, you lose some of the automagical creation of some documentation of the DAG task, but also lose the burdens that come with the "executing code to generate executable code" anti-pattern. Furthermore, any additional DAG tasks will then be able to use one of these operators instead of having to manually run some code to generate another Gusty configuration.